chia/wallet/conditions.py

```diff
@@ -427,7 +427,7 @@ def __post_init__(self) -> None:
             and self.coin_id_committed is None
             and self.mode_integer is None
         ):
-            raise ValueError("Must specify at least one committment. Anyone-can-send/recieve is not allowed.")
+            raise ValueError("Must specify at least one commitment. Anyone-can-send/receive is not allowed.")
```

Comment:
There is a test that checks for this specific wording, so if you change this exception string you have to update the test as well or CI will never pass